ios/Podfile.lock

```diff
@@ -13,7 +13,7 @@ PODS:
   - glog (0.3.5)
   - React (0.57.7):
     - React/Core (= 0.57.7)
-  - react-native-bpk-component-calendar (0.0.1):
+  - react-native-bpk-component-calendar (0.0.2):
```

Comment:
This seems related to previous release of calendar - how do we ensure we keep the demo app Podfile up-to-date?